goldenmaster/Plugins/TbSimple/Source/TbSimple/Private/Generated/Monitor/TbSimple.trace.cpp

```diff
@@ -188,13 +328,41 @@ void TbSimpleSimpleArrayInterfaceTracer::trace_callFuncInt(const TArray<int32>&
 	Tracer::instance()->call("tb.simple/SimpleArrayInterface#funcInt", fields_);
 }
 
+void TbSimpleSimpleArrayInterfaceTracer::trace_callFuncInt32(const TArray<int32>& ParamInt32)
+{
+	nlohmann::json fields_;
```

Comment:
Some inconsistencies in local scope variable naming, using trailing underscores  (e.g. fields_ vs most often not)

Reply:
Created issue for this https://github.com/apigear-io/template-unreal/issues/34